go/host/host.go

```diff
@@ -60,26 +60,24 @@ const (
 
 // Implementation of host.Host.
 type host struct {
-	config      config.HostConfig
-	shortID     uint64
-	isSequencer bool
+	config          config.HostConfig
+	shortID         uint64
+	isSequencer     bool
+	genesisRequired bool
```

Comment:
I've added a todo below to get rid of this `genesisRequired` flag but couldn't see a cleaner way to do it at the moment.

It's used to record "I am the genesis node and I haven't yet sent a genesis rollup" so that the genesis node knows to publish it once it's up-to-date in the main loop (there's no bootstrap anymore).